dashboard/controller/users.js

```diff
@@ -85,6 +85,7 @@ exports.addUser = function(req, res) {
 		req.body.color = JSON.parse(req.body.color);
 		req.assert('name', common.l10n.get('UsernameInvalid')).matches(/^[a-z0-9 ]+$/i);
 		req.assert('password', common.l10n.get('PasswordAtLeast', {count:ini.security.min_password_size})).len(ini.security.min_password_size);
+		req.assert('password', common.l10n.get('PasswordInvalid')).matches(/^[a-z0-9]+$/i);
```

Comment:
The whole list of characters is not alphanumeric. Have a look on login screen to see the full character set.